### PR TITLE
FIx: Rename action OpenSnsTokenSwap

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Users should not see the word Swap in the nns-dapp.

We want to show also the properties of the actions that are undefined. We show them as `NULL`.

# Changes

* Include `undefined` as the type of action field value that we support.
* Check when the action is OpenSnsTokenSwap and show a different title.

# Tests

* Add test for new support for `undefined` properties in action fields.
* Add test to check that ProposalProposerActionsEntry renders `NULL` for undefined properties.

### CODE REVIEW

File: frontend/src/lib/components/proposal-detail/ProposalProposerActionsEntry.svelte [hunk 1]
```diff
@@ -6,19 +6,25 @@
...
   import { KeyValuePair } from "@dfinity/gix-components";
+  import { i18n } from "$lib/stores/i18n";
 
   export let proposal: Proposal | undefined;
 
   let actionKey: string | undefined;
   let actionFields: [string, unknown][] = [];
   $: actionKey =
     proposal !== undefined ? proposalFirstActionKey(proposal) : undefined;
+  let title: string;
```
Comment (peterpeterparker) [inline frontend/src/lib/components/proposal-detail/ProposalProposerActionsEntry.svelte:17]: can you open a JIRA task and add a TODO to remove this hack once the NNS team has fixed the canisters?

File: frontend/src/lib/components/proposal-detail/ProposalProposerActionsEntry.svelte [hunk 2]
```diff
@@ -28,6 +34,8 @@
       <span class="value" slot="value">
         {#if typeof value === "object"}
           <Json json={value} />
+        {:else if typeof value === "undefined"}
+          {$i18n.core.null}
```
Comment (peterpeterparker) [inline frontend/src/lib/components/proposal-detail/ProposalProposerActionsEntry.svelte:38]: would be nice to have a test that checks that "NULL" is rendered in the Dom

**Review by peterpeterparker — APPROVED** [review]
Thanks. Two comments.

File: frontend/src/lib/components/proposal-detail/ProposalProposerActionsEntry.svelte (hunk 1 above)
Comment (lmuntaner) [reply frontend/src/lib/components/proposal-detail/ProposalProposerActionsEntry.svelte:18]: done

File: frontend/src/lib/components/proposal-detail/ProposalProposerActionsEntry.svelte (hunk 2 above)
Comment (lmuntaner) [reply frontend/src/lib/components/proposal-detail/ProposalProposerActionsEntry.svelte:39]: done
